CMakeLists.txt

```diff
@@ -70,6 +70,10 @@ option(WHISPER_ALL_WARNINGS_3RD_PARTY "whisper: enable all compiler warnings in
 option(WHISPER_FATAL_WARNINGS  "whisper: enable -Werror flag"               OFF)
 option(WHISPER_USE_SYSTEM_GGML "whisper: use system-installed GGML library" OFF)
 
+# flat bindings
+option(BINDINGS_FLAT      "Add extra flat definitions to Whisper + GGML" OFF)
```

Comment:
Perhaps this should be `WHISPER_BINDINGS_FLAT` to be consistent with the others?

Reply:
Yeah - I had that idea after deciding GGML_BINDINGS_FLAT was a good idea

Reply:
Be u0dating everything in a bit - in the middle of a huge all CUDA architecture build _ SYCL test (PCs going slow)